x-pack/plugins/cases/tsconfig.json

```diff
@@ -21,6 +21,7 @@
 
     // Required from './kibana.json'
     { "path": "../actions/tsconfig.json" },
+    { "path": "../rule_registry/tsconfig.json" },
```

Comment:
Without this I get ts errors saying we can't import files from the `rule_registry`.